resources/testharness.js

```diff
@@ -2383,13 +2383,54 @@ policies and contribution forms [3].
         return duplicates;
     };
 
+    function code_unit_str(char) {
+        return 'U+' + char.charCodeAt(0).toString(16);
+    }
+
+    function sanitize_unpaired_surrogates(str) {
+        return str.replace(/([\ud800-\udbff])(?![\udc00-\udfff])/g,
+                           function(_, unpaired)
+                           {
+                               return code_unit_str(unpaired);
+                           })
+                  // This replacement is intentionally implemented without a
```

Comment:
Understanding this commnet required me to go look up the fact that negative lookbheind is an experimental feature only shipping in V8. It might be worth saying that directly since "portability" immediately suggested "to non-web environments" and the difference between "we can't use this because currently not all browsers implement it" and "we can't use this because not all regexp engines implement it" is pretty significant to a future maintainer.

Reply:
Sounds good. Ideally, I'd like to reference a policy on this topic. Would you mind taking another look at gh-6333?

Reply:
@jgraham negative lookbehind is in ES2018 and not experimental

Reply:
OK, then I found an old document. All the more reason to have a more informative comment ;) (thanks for making the change!).